sass/_components.scss

```diff
@@ -1292,3 +1292,158 @@
         text-align: right;
     }
 }
+
+// Product Links Block
+//
+// default - Default Hero
+// :hover - Hover State
+//
+// markup:
+// <div class="product-links-block">
+//      <div class="columns is-vcentered has-centered-content">
+//          <div class="column has-centered-content">
+//              <div class="product-link product-link--tv {$modifiers}">
+//                  <a href="#">
+//                      <h3>Watch</h3>
+//                      <p>Real Vision</p>
+//                      <p><strong>Television</strong></p>
+//                      <div class="product-link__bottom-overlay">
+//                          <p class="product-link__bottom-arrow"></p>
+//                      </div>
+//                  </a>
+//              </div>
+//          </div>
+//          <div class="column has-centered-content">
+//              <div class="product-link product-link--publishing {$modifiers}">
+//                  <a href="#">
+//                      <h3>Read</h3>
+//                      <p>Real Vision</p>
+//                      <p><strong>Publishing</strong></p>
+//                      <div class="product-link__bottom-overlay">
+//                          <p class="product-link__bottom-arrow"></p>
+//                      </div>
+//                  </a>
+//              </div>
+//          </div>
+//          <div class="column has-centered-content">
+//              <div class="product-link product-link--podcast {$modifiers}">
+//                  <a href="#">
+//                      <h3>Listen</h3>
+//                      <p>Real Vision</p>
+//                      <p><strong>Podcast</strong></p>
+//                      <div class="product-link__bottom-overlay">
+//                          <p class="product-link__bottom-arrow"></p>
+//                      </div>
+//                  </a>
+//              </div>
+//          </div>
+//      </div>
+// </div>
+//
+// Styleguide 5.19
+.product-links-block {
+    background: url('#{$root}/images/components/product-links-block_bg_mobile.jpg');
+    background-size: cover;
+    padding: $baseline;
+    text-transform: uppercase;
+    color: white;
+    text-align: center;
+
+    @include tablet {
+        background: url('#{$root}/images/components/product-links-block_bg.jpg');
+    }
+
+    .product-link {
+        background: url('#{$root}/images/overlays/white_splatter.png'), #363e4e;
+        background-size: cover;
+        padding-top: 1rem;
+        padding-bottom: 3.5rem;
+        max-width: 275px;
```

Comment:
Should probably just rely on the column dimensions to restrict the size of this thing